### PR TITLE
entries: Bung 3rd Compo

### LINKED ISSUE #185: Bung Enterprises 3rd Coding Competition
This page refers to a 3rd coding competition (3rd news item from the top):

http://n64.icequake.net/mirror/www.bung.com.hk/html/news.htm

It notes the top entries as follows:

1st Prize : Apache
2nd Prize : IRONHIKE
3rd Prize : Cuberaid
3rd Prize : MIG 21

I'm unable to find a cache of the full entries/results lists, or the ROMs themselves.

### PR DESCRIPTION
**Pull request by dag7dev**
Luckily for us, the entire collection is available!

Close #185

### CODE REVIEW

File: entries/alpha-wing-bung-compo/20110402_alpha_wing.png
Comment (avivace) [inline entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: Why was this entry renamed?

Comment (dag7dev) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: on a First sight, the idea was to rename it to a more suitable name.

However I think just alpha-wing it's fine.

I will rename it in all lowercase.

Comment (avivace) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: The rename was Alpha-Wing -> alpha-wing-bung-compo

Comment (dag7dev) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: yup, I will rename it to Alpha-Wing -> alpha-wing so that it won't be messy

Comment (dag7dev) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: wouldn't we like to have lowercase names? while we wait to have an all-in-one renamer, we can safely rename by hand some of those entries if we work on it

Comment (avivace) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: Different tasks should be carried out in different commits, makes to sense to randomly start the renaming (of only one entry) here , together with a bunch of unrelated changes..

**Comment by dag7dev** [comment]
alrighty then, np

File: entries/alpha-wing-bung-compo/20110402_alpha_wing.png
Comment (dag7dev) [reply entries/alpha-wing-bung-compo/20110402_alpha_wing.png:1]: renamed back with the old name

**Comment by asiekierka** [comment]
https://github.com/gbdev/database/pull/223

**Pull request closed without merging**